bin/web-push.js

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env node
+const webPush = require('web-push');
```

Comment:
Actually I made it with that intention.

The use case is the following: I want to test my push notifications in the client and don't have the server side yet.

Solution:
You can do npm install -g web-push, and will provide you an executable to interact with the push service via command line

Does that make sense for you?